sdk/appcenter-data/src/test/java/com/microsoft/appcenter/data/UtilsTest.java

```diff
@@ -124,6 +124,15 @@ public void parseInvalidDate() {
         assertNull(document.getDeserializedValue());
     }
 
+    @Test
+    public void moduleHasNotStartedDoesNotThrow() {
+        Data.create("id", new DateDocument(), DateDocument.class, DefaultPartitions.APP_DOCUMENTS);
```

Comment:
check future exceptions

Reply:
Also this is the wrong test file to test that. Should be in DataTest.
You can Data.unsetInstance to unstart since the abstract setup does it.